code/modules/mob/living/carbon/carbon.dm

```diff
@@ -381,4 +381,42 @@
 		var/datum/gear/G = GLOB.gear_datums[i]
 		if(!G || !gear.Find(i))
 			continue
-		equip_to_slot_or_del(new G.path, SLOT_IN_BACKPACK)
\ No newline at end of file
+		equip_to_slot_or_del(new G.path, SLOT_IN_BACKPACK)
+
+
+
+/mob/living/carbon/human/update_sight()
+	if(!client)
+		return
+
+	if(stat == DEAD)
+		sight = (SEE_TURFS|SEE_MOBS|SEE_OBJS)
+		see_in_dark = 8
+		see_invisible = SEE_INVISIBLE_OBSERVER
+		return
+
+	sight = initial(sight)
+	lighting_alpha = initial(lighting_alpha)
+	see_in_dark = species.darksight
+	see_invisible = initial(see_invisible)
+
+	if(client.eye != src)
+		var/atom/A = client.eye
+		if(A.update_remote_sight(src)) //returns 1 if we override all other sight updates.
```

Comment:
```suggestion
		if(A?.update_remote_sight(src)) //returns 1 if we override all other sight updates.
```

Reply:
```
[21:25:47] Runtime in carbon.dm, line 405: Cannot execute null.update remote sight().
proc name: update sight (/mob/living/carbon/human/update_sight)
usr: Zyzara/(Oliver Kifer)
src: Oliver Kifer (/mob/living/carbon/human)
src.loc: null
call stack:
Oliver Kifer (/mob/living/carbon/human): update sight()
Oliver Kifer (/mob/living/carbon/human): reset perspective(null)
Oliver Kifer (/mob/living/carbon/human): Login()
Oliver Kifer (/mob/living/carbon/human): Login()
Oliver Kifer (/mob/living/carbon/human): Login()
/datum/mind (/datum/mind): transfer to(Oliver Kifer (/mob/living/carbon/human), 1)
Distress Signal (/datum/game_mode/distress): end of round deathmatch()
Distress Signal (/datum/game_mode/distress): declare completion(0)
Distress Signal (/datum/game_mode/distress): declare completion(0)
Ticker (/datum/controller/subsystem/ticker): fire(0)
Ticker (/datum/controller/subsystem/ticker): ignite(0)
```

Reply:
That would be supressing a runtime. Client eye should not be null.